src/group_theory/complement.lean

```diff
@@ -360,6 +360,9 @@ h.is_compl.sup_eq_top
 lemma is_complement'.disjoint (h : is_complement' H K) : disjoint H K :=
 h.is_compl.disjoint
 
+lemma is_complement'.index_eq_card (h : is_complement' H K) : K.index = nat.card H :=
+nat.card_congr (equiv.of_bijective _ $ mem_left_transversals_iff_bijective.1 h).symm
```

Comment:
The same proof works when `H` is not necessarily a subgroup:
```
lemma is_complement.index_eq_card (h : is_complement T K) : K.index = nat.card T :=
nat.card_congr (equiv.of_bijective _ $ mem_left_transversals_iff_bijective.1 h).symm
```
but maybe this should be stated in terms of `transversals`?